packages/geo/src/lib/print/shared/print.service.ts

```diff
@@ -1019,6 +1032,10 @@ export class PrintService {
         }
       }
     });
+    // Set map image size
+    map.ol.setSize(newMapSize);
+    const scaling = Math.min(newMapSize[0] / initialMapSize[0], newMapSize[1] / initialMapSize[1]);
+    map.ol.getView().setResolution(viewResolution / scaling);
```

Comment:
Pour éviter de laisser des commentaire on pourrait en faire un méthode, cela améliorait aussi la lisibilité.
```suggestion 
    setMapResolution(initialSize: [number, number], resolution: PrintResolution): void {
        const scaleFactor = resolution / 96;
        const [width, height] = initialSize;

        const scaledSize = [Math.round(width * scaleFactor), Math.round(height * scaleFactor)];
        map.ol.setSize(scaledSize);
        
        const scaling = Math.min(scaledSize[0] / width, scaledSize[1] / height);
        const view = map.ol.getView();
        view.setResolution(view.getResolution() / scaling);
    }
```